mlrun/api/crud/model_monitoring/model_endpoint_store.py

```diff
@@ -468,34 +468,52 @@ def delete_model_endpoints_resources(
             container=self.container,
         )
 
-        # Getting the path for the time series DB
-        events_path = (
-            mlrun.mlconf.model_endpoint_monitoring.store_prefixes.default.format(
-                project=self.project,
-                kind=mlrun.api.schemas.ModelMonitoringStoreKinds.EVENTS,
-            )
-        )
-        (
-            _,
-            _,
-            events_path,
-        ) = mlrun.utils.model_monitoring.parse_model_endpoint_store_prefix(events_path)
+        # Generate the required tsdb paths
+        tsdb_path, filtered_path = self._generate_tsdb_paths()
 
         # Delete time series DB resources
         try:
             frames.delete(
                 backend=model_monitoring_constants.StoreTarget.TSDB,
-                table=events_path,
+                table=filtered_path,
                 if_missing=v3io_frames.frames_pb2.IGNORE,
             )
         except v3io_frames.errors.CreateError:
             # Frames might raise an exception if schema file does not exist.
             pass
 
         # Final cleanup of tsdb path
-        events_path.replace("://u", ":///u")
-        store, _ = mlrun.store_manager.get_or_create_store(events_path)
-        store.rm(events_path, recursive=True)
+        tsdb_path.replace("://u", ":///u")
+        store, _ = mlrun.store_manager.get_or_create_store(tsdb_path)
+        store.rm(tsdb_path, recursive=True)
+
+    def _generate_tsdb_paths(self) -> typing.Tuple[str, str]:
+        """Generate a short path to the TSDB resources and a filtered path for the frames object
```

Comment:
```suggestion
        """
        Generate a short path to the TSDB resources and a filtered path for the frames object
```

Reply:
One of the TSDB path parsing steps in `delete_model_endpoints_resources()` function was removed in part 1 of the SQL abstraction PR: https://github.com/mlrun/mlrun/pull/2378/. In this PR I brought it back (along with tests and code structure improvements).